src/pages/index.js

```diff
@@ -1,55 +1,74 @@
-import Head from 'next/head'
-import styles from '../styles/Home.module.css'
+import { useEffect, useState } from "react";
+import firebase from "firebase/app";
+import "firebase/auth";
+import "firebase/firestore";
+import ChatRoom from "../Components/ChatRoom";
```

Comment:
you named chatroom file as Chatroom.js, will it work here?

Reply:
yes that is always working. i also doing previously like this